src/Components/TicketSelector/SeatOption/SeatOption.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useRef } from "react";
+import styled from "styled-components";
+import { VscChromeClose } from "react-icons/vsc";
+import { AiOutlineMinus, AiOutlinePlus } from "react-icons/ai";
+import { useClickOutside } from "../../Hooks/clickOutside";
+import { theme, flexSet } from "../../../styles/theme";
+import { SEAT_OPTIONS } from "./seatoptionData";
+
+const SeatOption = ({ seatType, headCount, handleRadioChange, handleHeadCount, closeSeatOption }) => {
+  const seatOptionRef = useRef(null);
+
+  const handleChange = (option) => {
+    handleRadioChange(option);
+  };
+
+  const handleMinusClick = () => {
+    handleHeadCount("minus");
+  };
+
+  const handlePlusClick = () => {
+    handleHeadCount("plus");
+  };
+
+  const closeSeat = () => {
+    closeSeatOption();
+  };
+
+  useClickOutside(seatOptionRef, closeSeatOption);
+
+  return (
+    <SeatOptionContainer ref={seatOptionRef}>
+      <header>
+        <span>인원 & 좌석등급</span>
+        <button onClick={closeSeat}>
+          <VscChromeClose size="20px" color={"rgba(0,0,0,0.5)"} />
+        </button>
+      </header>
+      <PeopleSelector>
+        <div className="peopleType">
+          <p>성인</p>
+          <span>만 12세 이상</span>
+        </div>
+        <div className="peopleCounter">
+          <button className="minusButton" name="minus" onClick={handleMinusClick}>
```

Comment:
함수 컴포넌트에서는 렌더와 함께 어차피 함수가 재생성되기 때문에 가독성 때문에라도 즉시 실행함수로 써주는게 좋을 것 같습니다.
```suggestion
          <button className="minusButton" name="minus" onClick={() => handleHeadCount("minus")}>
```

Reply:
수정하겠습니다!!